Lib/test/datetimetester.py

```diff
@@ -266,7 +266,8 @@ def test_repr(self):
             self.assertEqual(tz, eval(tzrep))
 
     def test_class_members(self):
-        limit = timedelta(hours=23, minutes=59)
+        limit = timedelta(hours=23, minutes=59,
```

Comment:
In addition to my other comments, I guess this comment got lost. Please leave this value intact as `timedelta(hours=23, minutes=59)` and leave `datetime.py` intact. You can do this by changing [the definition of `timezone.(min|max)`](https://github.com/python/cpython/blob/717cccfbac474de580263ae93a1c75f37c1033ed/Lib/datetime.py#L2297) like this:

```python
timezone.utc = timezone._create(timedelta(0))
# bpo-37642: These attributes are rounded to the nearest minute for backwards
# compatibility, even though the constructor will accept a wider range of
# values. This may change in the future.
timezone.min = timezone._create(-timedelta(hours=23, minutes=59))
timezone.max = timezone._create(timedelta(hours=23, minutes=59))
_EPOCH = datetime(1970, 1, 1, tzinfo=timezone.utc)
```

Note the comment. If you want you can put a similar comment in `_datetimemodule.c`.